_component_design/breadcrumbs.md

```diff
@@ -66,20 +63,20 @@ Sometimes we can get so deep in an interface we lose track of where we are. Whil
 {% endcolumn %}
 </div>
 
-
 <div class="row">
 {% column left:"col-md-4" %}
 ## To many items
 
-* __Jump The Crumb:__ Maybe we’ve really overdone it on the architecture. It happens (does it have to though?). Whattaya do? If you can’t abbreviate adequately, we have to jump the breadcrumbs. What’s that mean?
+<!---* __Jump the crumb:__ Maybe we’ve really overdone it on the architecture. It happens (does it have to though?). Whattaya do? If you can’t abbreviate adequately, we have to jump the breadcrumbs. What’s that mean?--->
 
-* __What to Keep:__ Keep the first item in the architecture so people can get back to the top level to get to homebase and re-orient, then keep the last three items in your architecture, including the page you are currently on. The ellipses, as of now, is not clickable / expandable.
+* __What to Keep:__ Keep the first item in the architecture so it is easy to navigate, re-orientate, and return to home. Keep the last three items in the architecture, including the current page. The ellipses are not clickable or expandable.
+
+* __Still too long?:__ We recommend only including the last two items in the architecture, inclusive of the current page.
 
-* __Still Too Long?:__ If you need to, and use your judgment here, only include the last two items in the architecture, inclusive of the page you are on. __DO NOTE__ only keep the first item and the last item. Doesn’t do the user much good.
```

Comment:
Typo on line 78, __DO NOTE__ should be __DO NOT__

Reply:
The red is what is removed :) the green is what I have left. So this typo does not exist :)